src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -33,7 +33,10 @@
     private List<String> readFromAMCaseTypes;
 
     @Value("#{'${ccd.callback.retries}'.split(',')}")
-    private List<Integer> callbackRetries;
+    private List<Integer> callbackRetryIntervalsInSeconds;
```

Comment:
this new name could be confused with the interval of time in seconds between callbacks, which is something different. Can we call it `callbackDurationInSeconds`?

Reply:
This actually is the interval of time in seconds between callbacks.

Reply:
ok I see. Then we should rename `ccd.callback.retries` as well?

Reply:
Sure changed now. Ta